### ISSUE TITLE
checkV download broken

### ISSUE THREAD

**Issue by gbouras13** [linsalrob/sphae#28]
Installing sphae on a new machine.

I got an error with `sphae install` - checkV download issue.

Looks like the issue is known https://bitbucket.org/berkeleylab/checkv/issues/108/issue-when-downloading-checkv-db

Manual solution suggested until the developers solve it.

```
# download the tarball
wget https://portal.nersc.gov/CheckV/checkv-db-v1.5.tar.gz
# extract tarball
tar xvf checkv-db-v1.5.tar.gz
# make diamond db
diamond makedb \
  --in ${PWD}/checkv-db-v1.5/genome_db/checkv_reps.faa \
  --db ${PWD}/checkv-db-v1.5/genome_db/checkv_reps \
  --threads 1 \
  1> /dev/null \
  2> ${PWD}/checkv-db-v1.5/genome_db/checkv_reps.log 
```

George

**Comment by npbhavya** [comment]
Looks like this issue was fixed on checkv end. I am now able to download checkv database with `sphae install` command. But I was running into another error when running `sphae run` command. 

CheckV v1.0.1: completeness
[1/8] Skipping gene calling...
[2/8] Initializing queries and database...
[3/8] Running DIAMOND blastp search...
/bin/sh: line 1: 401169 Segmentation fault      (core dumped) diamond blastp --outfmt 6 --evalue 1e-5 --query-cover 50 --subject-cover 50 -k 10000 --query example/PROCESSING/assembly/flye/nanoreads-sr/checkv/tmp/proteins.faa --db /scratch/user/nala0006/spae-paper/sphae/sphae/workflow/databases/checkv-db-v1.5/genome_db/checkv_reps.dmnd --threads 1 > example/PROCESSING/assembly/flye/nanoreads-sr/checkv/tmp/diamond.tsv 2> example/PROCESSING/assembly/flye/nanoreads-sr/checkv/tmp/diamond.tsv.log
DIAMOND task failed. Program should be rerun.
================================================================================

Exiting because a job execution failed. Look above for error message

Fixed this by reverting to diamond v2.18 in the yaml file. I will push this version soon along with other updates.

**Comment by npbhavya** [comment]
Resolved in v1.3.3
